service/raft/raft_rpc.cc

```diff
@@ -5,20 +5,26 @@
 /*
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
-#include <seastar/core/coroutine.hh>
 #include "service/raft/raft_rpc.hh"
+#include <seastar/core/coroutine.hh>
 #include "gms/inet_address.hh"
 #include "gms/inet_address_serializer.hh"
 #include "serializer_impl.hh"
 #include "message/msg_addr.hh"
 #include "message/messaging_service.hh"
 #include "db/timeout_clock.hh"
 #include "idl/raft.dist.hh"
+#include "service/raft/raft_address_map.hh"
+#include "service/raft/raft_state_machine.hh"
 
 namespace service {
 
 static seastar::logger rlogger("raft_rpc");
 
+raft_ticker_type::time_point timeout() {
+    return raft_ticker_type::clock::now() + raft_tick_interval * (raft::ELECTION_TIMEOUT.count() / 2);
+}
+
```

Comment:
This move seems unrelated.

Reply:
In what way? I moved the function from the header file to a C file, where it's used, because hte header file is not containing raft_ticker_type any more.

Reply:
Ah, ok, it's fine then.